efficient_full_table_scan_example_code/README.md

```diff
@@ -4,13 +4,13 @@ In this example we demonstrate an efficient parallel full table scan, that utili
 to gain servers and cores parallelism optimization.
 
 Following setup was used in this example:
-- Scylla cluster v1.6.1: 3 x RHEL7.2 nodes deployed on Google Compute Engine (GCE), each has 8 vCPUs and 30GB memory
+- Scylla cluster v4.3: 3 x RHEL7.2 nodes deployed on Google Compute Engine (GCE), each has 8 vCPUs and 30GB memory
 - Storage: each node has 2 x 375GB local SSDs (NVMe) disks  
-- Client: Centos7.2 node deployed on GCE (4 vCPU and 15GB memory) will run our golang code using gocql driver
+- Client: Centos7.2 node deployed on GCE (4 vCPU and 15GB memory) will run our golang code using gocql driver or Ubuntu
 - Client prerequisites:
-	- Install go 1.7: https://www.digitalocean.com/community/tutorials/how-to-install-go-1-7-on-centos-7
+	- Install go 1.20 * (see instructions below for local install, or use your OS package manager to get recent go)
 	- Install gocql driver: "go get github.com/gocql/gocql"
-	- Install kingpin pkg: "go get gopkg.in/alecthomas/kingpin.v2"
+	- Install kingpin pkg: "go get github.com/alecthomas/kingpin/v2"
```

Comment:
I suggest changing it to `go install` instead of `go get` (should the users user latest or a specific version)?
```
[guy@fedora efficient_full_table_scan_example_code]$ go get github.com/gocql/gocql
go: go.mod file not found in current directory or any parent directory.
	'go get' is no longer supported outside a module.
	To build and install a command, use 'go install' with a version,
	like 'go install example.com/cmd@latest'

```